uber/models.py

```diff
@@ -1052,6 +1052,12 @@ class Group(MagModel, TakesPaymentMixin):
     name            = Column(UnicodeText)
     tables          = Column(Numeric, default=0)
     address         = Column(UnicodeText)
```

Comment:
Do we want to keep the `address` column?

This migration may prove a bit trickier than we initially thought. If we decide we don't want the `address` column, then we probably want the migration to run an `ALTER` statement that renames `address` to `address1`.

Also, comparing the group address fields with the attendee address fields, I noticed attendee also has an international field:
```python
international = Column(Boolean, default=False)
```

Do we need/want that for groups as well? I'm not familiar with the original use-case for the international checkbox. Was that just an easy way to bypass the address requirements for attendee registration? Because we definitely can't let dealers bypass the address requirements.